src/css/custom.css

```diff
@@ -382,133 +101,111 @@ html[data-theme='dark'] {
+  margin-left: 2rem;
+}
 
-  .section-header {
-    font-family: 'Inter';
-    font-weight: var(--ifm-font-weight-semibold);
-    font-size: 1.5rem;
-    letter-spacing: 0.04em;
-    color: #909FBE;
-  }
+.navbar__brand {
+  height: 25px;
+}
 
-  [data-theme='dark']  .section-header {
-   color: #B0BFD9;
-  }
+.section-header {
+  font-family: 'Inter';
+  font-weight: var(--ifm-font-weight-semibold);
+  font-size: 1.5rem;
+  letter-spacing: 0.04em;
+  color: var(--ifm-color-gray-500);
+}
 
-  .footer {
-    border-top: 1px solid #E8EEFB;
-  }
+[data-theme='dark']  .section-header {
+  color: var(--ifm-color-gray-400);
+}
 
-  [data-theme='dark'] .footer {
-    border-top: 1px solid var(--ifm-color-gray-700);
-  }
+.footer {
+  border-top: 1px solid var(--ifm-color-gray-300);
+}
 
+[data-theme='dark'] .footer {
+  border-top: 1px solid var(--ifm-color-gray-800);
+}
 
-  .footer__links {
-    margin-left: max(1rem, calc(20% - 5rem));
-    margin-right: max(1rem, calc(20% - 5rem));
-    font-size: 1rem;
-  }
 
-  .footer__bottom {
-    display: flex;
-    align-items: flex-end;
-    flex-wrap: wrap;
-    margin-left: max(1rem, calc(20% - 5rem));
-    margin-right: max(1rem, calc(20% - 5rem));
-  }
+.footer__links {
+  margin-left: max(1rem, calc(20% - 5rem));
+  margin-right: max(1rem, calc(20% - 5rem));
+  font-size: 1rem;
+}
 
-  .footer__logo {
-    height: 3rem;
-    margin-right: 1em;
-    margin-bottom: -1em;
-  }
+.footer__bottom {
+  display: flex;
+  align-items: flex-end;
+  flex-wrap: wrap;
+  margin-left: max(1rem, calc(20% - 5rem));
+  margin-right: max(1rem, calc(20% - 5rem));
+}
 
-  .footer__copyright {
-    font-size: 0.9rem;
-    margin-top: 1em;
-  }
+.footer__logo {
+  height: 3rem;
+  margin-right: 1em;
+  margin-bottom: -1em;
+}
 
-  ::selection {
-    background-color: #b3d4fc;
-    color: #000;
-    text-shadow: none;
-  }
+.footer__copyright {
+  font-size: 0.9rem;
+  margin-top: 1em;
+}
 
-  .grey {
-    color: var(--ifm-navbar-link-color);
-  }
+::selection {
+  background-color: #b3d4fc;
+  color: #000;
+  text-shadow: none;
+}
 
-  .headline-stick {
-    height: 4px;
-    background-color: var(--ifm-color-primary);
-    margin-bottom: 1.25rem;
-    width: 16px;
-    transition: all var(--ifm-transition-fast) var(--ifm-transition-timing-default);
-  }
+.stick {
+  height: 4px;
+  background-color: var(--ifm-color-primary);
+  margin-bottom: 1.25rem;
+  width: 16px;
+  transition: all var(--ifm-transition-fast) var(--ifm-transition-timing-default);
+}
 
-  .headline-stick.size-s {
-    width: 32px;
-  }
+.stick--sm {
+  width: 32px;
+}
 
-  .headline-stick.size-m {
-    width: 48px;
-  }
+.stick--md {
+  width: 48px;
+}
 
-  .hash-link:hover {
-    border: none;
-  }
+.hash-link:hover {
+  border: none;
+}
 
-  /*
-  Patch to avoid displaying a double header in imported DOCS
-  */
-  div.markdown > p > header > h1 {
+/*
+Patch to avoid displaying a double header in imported DOCS
+*/
+.markdown header h1 {
```

Comment:
This change removes every h1 headline in the docs. That was not the idea?

Reply:
Ah excuse me, got a little too enthusiastic there I think!